src/giftCards/GiftCardsList/providers/GiftCardListProvider/GiftCardListProvider.tsx

```diff
@@ -65,23 +80,37 @@ export const GiftCardsListProvider: React.FC<GiftCardsListProviderProps> = ({
 
   const paginationState = createPaginationState(settings.rowNumber, params);
 
+  const handleSort = createSortHandler(navigate, giftCardListUrl, params);
+
   const queryVariables = React.useMemo<GiftCardListVariables>(
     () => ({
       ...paginationState,
-      filter: getFilterVariables(params)
+      filter: getFilterVariables(params),
+      sort: getSortQueryVariables(params)
     }),
-    [params]
+    [params, settings.rowNumber]
   );
 
+  const handleGiftCardListError = (error: ApolloError) => {
+    const { message } = error.graphQLErrors[0];
+    notify({
+      status: "error",
+      text: message
+    });
+  };
```

Comment:
I don't think we should be handling errors in backend that originate from not out incorrect input but (probably) bug in application. What errors can we expect here?

Reply:
Backend sends an error message that it is required to have a currently filter applird to sort by balance. For now there's no other place to add error handling like that.

Reply:
@dominik-zeglen Do you recommend a different approach to this problem?

Reply:
Not really, lets leave it here